strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetArtifactController.java

```diff
@@ -459,7 +434,7 @@ public ResponseEntity putPackage(@RequestHeader(name = "X-NuGet-ApiKey", require
         try
         {
             ServletInputStream is = request.getInputStream();
-            FileInputStream packagePartInputStream = extractPackageMultipartStream(extractBoundary(contentType), is);
+            InputStream packagePartInputStream = extractPackageMultipartStream(extractBoundary(contentType), is);
```

Comment:
+1: Yeah, we should use interfaces, instead of implementations (wherever possible)! :)